frontend/src/components/dashboard/subdomain/SubdomainInfoTooltip.tsx

```diff
@@ -24,17 +24,35 @@ import styles from "./SubdomainInfoTooltip.module.scss";
 import { Localized } from "../../Localized";
 import { InfoModal } from "../../InfoModal";
 
+export type Props = {
```

Comment:
Q: should this exported `Props` have a more unique name?
We already have a `MobileExplainerModalProps` type below, so not sure if it's potentially vague and confusing.

Reply:
Renamed to `SubdomainInfoTooltipProps`